templates/default/etc/init.d/knotx.erb

```diff
@@ -2,62 +2,62 @@
 
 #
 # chkconfig: - 80 20
-# processname: <%= @knotx_id %>
+# processname: <%= @id %>
 # description: Knotx - reactive microservice assembler
 #
 
 ### BEGIN INIT INFO
-# Provides: <%= @knotx_id %>
+# Provides: <%= @id %>
 # Required-Start: $network $syslog
 # Required-Stop: $network $syslog
 # Default-Start:
 # Default-Stop:
-# Short-Description: start and stop <%= @knotx_id %>
+# Short-Description: start and stop <%= @id %>
 ### END INIT INFO
 
 . /etc/rc.d/init.d/functions
 
-CONFIG_FILE="<%= @knotx_root_dir %>/knotx.conf"
+CONFIG_FILE="<%= @home_dir %>/knotx.conf"
 
 [ -e $CONFIG_FILE ] && . $CONFIG_FILE
 
 start()
 {
-  echo -n $"Starting <%= @knotx_id %>: "
-  runuser -l "<%= @knotx_user %>" -c "java \
-  -Dlogback.configurationFile=<%= @knotx_root_dir %>/logback.xml \
-  -Dvertx.cacheDirBase=<%= @knotx_root_dir %>/.vertx \
+  echo -n $"Starting <%= @id %>: "
+  runuser -l "<%= @user %>" -c "<%= @java_home %>/bin/java \
+  ${KNOTX_GENERAL_OPTS} \
+  ${KNOTX_GC_OPTS} \
+  ${KNOTX_JMX_OPTS} \
+  ${KNOTX_DEBUG_OPTS} \
   -Xms${KNOTX_MIN_HEAP}m \
   -Xmx${KNOTX_MAX_HEAP}m \
-  -XX:ReservedCodeCacheSize=${KNOTX_CODE_CACHE}m \
   -XX:+UseBiasedLocking \
   -XX:BiasedLockingStartupDelay=0 \
-  ${KNOTX_GENERAL_OPTS} \
-  ${KNOTX_JMX_OPTS} \
-  ${KNOTX_DEBUG_OPTS} \
-  ${KNOTX_GC_OPTS} \
-  ${KNOTX_EXTRA_OPTS} \
-  -Dio.knotx.KnotxServer.options.config.serverOptions.port=${KNOTX_PORT} \
-  -cp \"<%= @knotx_root_dir %>/app/*\" io.knotx.launcher.LogbackLauncher \
-  ${KNOTX_CONFIG} \
-  ${KNOTX_CONFIG_EXTRA} >> /dev/null 2>&1 &" && echo_success || echo_failure
+  -Dlogback.configurationFile=<%= ::File.join(@conf_dir, 'logback.xml') %> \
+  -Dvertx.logger-delegate-factory-class-name=io.vertx.core.logging.SLF4JLogDelegateFactory \
+  -Dknotx.home=<%= @home_dir %> \
+  -Dvertx.cacheDirBase=<%= ::File.join(@home_dir, '.vertx' ) %> \
+  -Dvertx.cli.usage.prefix=knotx \
+  -classpath ':<%= @conf_dir %>:<%= ::File.join(@lib_dir, '*') %>' \
+  $KNOTX_EXTRA_OPTS \
+  io.vertx.core.Launcher run-knotx >> /dev/null 2>&1 &" && echo_success || echo_failure
```

Comment:
btw, the stdout still prints out useful messages in this case about logs rollout

Reply:
Fixed. `stdout` got redirected to `$LOG_DIR/start.log`